frontend/src/pages/modelServing/screens/projects/nimUtils.ts

```diff
@@ -221,9 +214,9 @@ export const fetchNIMAccountConstants = async (
     }
 
     return {
-      nimSecretName,
-      nimNGCSecretName: nimPullSecret.name,
-      nimConfigMapName: nimConfig.name,
+      nimSecretKey: 'apiKeySecret',
+      nimNGCSecretKey: 'nimPullSecret',
+      nimConfigMapKey: 'nimConfig',
       templateName: runtimeTemplate.name,
     };
```

Comment:
The return data structure seems a little confusing.  The name of the function indicates it is fetching data, but 3/4 of the values are hardcoded. And 1 is not, but they are all the same type of member of the nimAccount. 

So it looks like the first 3 are just used to eventually get passed into the backend api which has conditional logic depending on these. But the templateName can be used directly by the frontend for checking resources.

Reply:
Updated